lib/Notifications.dart

```diff
@@ -0,0 +1,208 @@
+import 'package:flutter/material.dart';
+import 'Home.dart';
+
+class Notifications extends StatefulWidget {
+  Notifications({Key key}) : super(key: key);
+
+  @override
+  _NotificationsState createState() => _NotificationsState();
+}
+
+class _NotificationsState extends State<Notifications> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          'Schedule',
+          style:
+              TextStyle(color: Colors.black, fontSize: 20, fontFamily: 'SFPro'),
+        ),
+        backgroundColor: Colors.white,
+        titleSpacing: 0.0,
+        iconTheme: IconThemeData(color: Colors.black),
+        automaticallyImplyLeading: true,
+        leading: IconButton(
+          icon: Icon(Icons.arrow_back_ios),
+          onPressed: () => Home(),
+        ),
+      ),
+      body: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Padding(
+              padding: EdgeInsets.only(left: 24.0, top: 10.0, bottom: 8.0),
+              child: Text('Notifications',
+                  style: Theme.of(context).textTheme.headline),
+            ),
+            Detail(),
+            Container(
+              color: Colors.white,
+              height: MediaQuery.of(context).size.height / 3,
+            )
+          ]),
+    );
+  }
+}
+
+class Detail extends StatefulWidget {
+  @override
+  _DetailState createState() => _DetailState();
+}
+
+class _DetailState extends State<Detail> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        decoration: BoxDecoration(
+          color: Colors.white,
+          borderRadius: BorderRadius.circular(3),
+          boxShadow: [
+            BoxShadow(
+                color: Color.fromARGB(15, 0, 0, 0),
+                offset: Offset(0, 4.0),
+                blurRadius: 10.0,
+                spreadRadius: 1.0)
+          ],
+        ),
+        child: Padding(
+            padding: EdgeInsets.only(top: 24, left: 16, right: 16),
+            child: Column(
+              children: <Widget>[
+                Row(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    CircleAvatar(
+                      radius: 22,
+                      child: CircleAvatar(
+                        radius: 22,
+                        backgroundColor: Colors.black,
+                        foregroundColor: Colors.white,
+                        child: Icon(Icons.directions_car),
+                      ),
+                    ),
+                    SizedBox(width: 19),
+                    Expanded(
+                      child: Text(
+                        "Your driver is on the way! Wait outside to meet your driver.",
+                        style: TextStyle(fontSize: 15, color: Colors.black),
+                      ),
+                    ),
+                  ],
+                ),
```

Comment:
This structure seems to be used a lot. Maybe factor it out into a separate stateless widget that has some arguments controlling things such as the color and the message.